tests/e2e/network/service_annotations.go

```diff
@@ -315,3 +346,57 @@ func validateInternalLoadBalancer(c clientset.Interface, ns string, url string)
 	utils.Logf("validation finished")
 	return err
 }
+
+func validateLoadBalancerBackendPools(vmssname string, cs clientset.Interface, labels map[string]string, ns string, ports []v1.ServicePort, resourcegrpverify string) {
+	// create deployment
+	serviceName := "validate-lb-mode"
+	lbModeLoadBalancerName := serviceName + string(uuid.NewUUID())
+
+	utils.Logf("Creating deployment " + lbModeLoadBalancerName)
+	deployment := createNginxDeploymentManifest(lbModeLoadBalancerName, labels)
+
+	// create service annotation mode for LoadBalancer
+	mode_name := vmssname
+	annotation := map[string]string{
+		azure.ServiceAnnotationLoadBalancerMode: mode_name,
```

Comment:
actually `mode_name` is not needed, `vmssname` could be used here directly.

Reply:
okk Sir